templates/search/catalogue_search.html

```diff
@@ -23,7 +23,8 @@
 {% endblock %}
 
 {% block content %}
-    {{ form.non_field_errors }}
+    {# commented for future requirements #}
+    {% comment %} {{ form.non_field_errors }} {% endcomment %}
```

Comment:
We shouldn't be doing anything with non-field errors here. Why is this change needed?

Reply:
Same comment as above on non -field errors and mention in the ticket.

Reply:
Like I've said elsewhere, errors relating to field input are field errors. We shouldn't be changing what non field errors are doing.

Reply:
@ababic isn't a combination of fields a non field error?

> Same comment as above on non -field errors and mention in the ticket.

> Currently Date validation error - "There is a problem. Start date cannot be after end date." is tagged to a field opening_start_date, whereas this is a non field error. Moved it to non field error. The error is not specific to a particular field, but a set of fields.

> We have non field errors displayed at the top of the page (commented out for now), whereas we want the user to to views those errors at the Filter location. 

Would you like to take this discussion in the ticket?